src/evm/mod.rs

```diff
@@ -1054,6 +1054,33 @@ pub fn evm_direct_deploy_add(log_to: Option<&Path>, debug: bool) {
     }
 }
 
+pub fn _evm_test_extcodesize_of_constructor(log_to: Option<&Path>) {
```

Comment:
I would recommend placing this within a test file so the compiler won't ignore it if it becomes unused.

Reply:
Changed this to #[cfg(test)] and removed initial underscore from its name.